distribution/resources/conf/micro-gw-default.conf

```diff
@@ -1,30 +1,55 @@
+# Transport listener Configurations
 [listenerConfig]
-  host="0.0.0.0"
-  httpPort=9090
-  httpsPort=9095
-  keyStorePath="${ballerina.home}/bre/security/ballerinaKeystore.p12"
-  keyStorePassword="ballerina"
-  trustStorePath="${ballerina.home}/bre/security/ballerinaTruststore.p12"
-  trustStorePassword="ballerina"
-  tokenListenerPort=9096
-
+  # Microgateway exposed IP / Host
+  host = "0.0.0.0"
+  # HTTP port that is used to make APIs available to the outside.
+  httpPort = 9090
+  # HTTPS port that is used to make APIs available to the outside and for endpoints(/token, /authorize, /revoke/, userinfo) of Key Manager.
+  httpsPort = 9095
+  # HTTP port for endpoints(/token, /authorize, /revoke/, userinfo) of Key Manager
+  tokenListenerPort = 9096
+  # Internal keystore
+  keyStorePath = "${ballerina.home}/bre/security/ballerinaKeystore.p12"
+  keyStorePassword = "ballerina"
+  # Truststore
+  trustStorePath = "${ballerina.home}/bre/security/ballerinaTruststore.p12"
+  trustStorePassword = "ballerina"
+
+# API Authorization security for backend
 [authConfig]
-  authorizationHeader="Authorization"
-  removeAuthHeaderFromOutMessage=true
+  # Authorization header
```

Comment:
Global Authorization header expected by gateway. Can be override at API level

Reply:
Thanks. Fixed in 3c15c88